### ISSUE TITLE
Webworkflow code editor not working

### ISSUE THREAD

**Issue by casainho** [circuitpython/web-editor#90]
### CircuitPython Version

```python
Adafruit CircuitPython 8.1.0-beta.2, ESP32-S3-DevKitC-1-N8R2 board.
```

### Workflow(s)

WEB

### Browser(s)

```python
Firefox and Chrome, Linux, Android and MAC.
```

### Steps

http://cpy-2b4f34.local/code/ don't load the code editor. Some weeks ago it was working perfectly for me.

### Description

When the code page opens, I quickly see the following string: "Unable to connect. The device [local IP address] was not found. Be sure it is plugged in and set up properly."
Then I see this:

![235932207-1b572b57-89f6-4442-86bc-0624d6dabd7a](https://user-images.githubusercontent.com/2681417/235963388-0cc9662f-9dde-4ace-81e9-dbf27e21b5bb.png)

I erased and flashed a few times the CircuitPython binary.

I tested to change the CIRCUITPY_WEB_API_PORT.

I tested in two different ESP32-S3 as also in one ESP32-S2, that always worked well for me.

I also tested on a different wifi network, with only my PC and the ESP32-S3 board.

Tested on 2 different Linux Ubuntu computers, on one Android Phone and on one MAC.

### Additional information

See https://github.com/adafruit/circuitpython/issues/7929

**Comment by makermelissa** [comment]
This was likely caused by #77. I'll take a look at it.

**Comment by makermelissa** [comment]
Ok, I was able to reproduce the issue. Coincidentally, the only board I was able to find happened to be the same one you have. I'm almost certain I was correct about the cause of the issue now.

**Comment by makermelissa** [comment]
Here's the console output:

```
index.css:1     Failed to load resource: the server responded with a status of 404 (Not Found)
loading-blinka.webp:1     Failed to load resource: the server responded with a status of 404 (Not Found)
index.js:1     Failed to load resource: the server responded with a status of 404 (Not Found)
index.css:1     Failed to load resource: the server responded with a status of 404 (Not Found)
```

**Comment by makermelissa** [comment]
Fixed by #91.

**Comment by casainho** [comment]
Works as expected, thank you!! :+1: :partying_face: 

![Screenshot from 2023-05-03 18-45-09](https://user-images.githubusercontent.com/147128/236001375-7ce45c66-0393-4f50-b4df-0e867c00e016.png)

![Screenshot from 2023-05-03 18-45-24](https://user-images.githubusercontent.com/147128/236001368-c219f630-11a5-4c50-b7ce-9d8a1ccb1862.png)

![Screenshot from 2023-05-03 18-45-54](https://user-images.githubusercontent.com/147128/236001360-cbadcd89-81f1-4c7a-acbf-c21f16dd9f2a.png)
